schutzbot/Jenkinsfile

```diff
@@ -814,58 +814,60 @@ void run_tests(test_type) {
     }
 
     if (test_type == 'integration') {
+        sh (
+            label: "Satellite test",
+            script: "/usr/libexec/tests/osbuild-composer/satellite.sh"
+        )
```

Comment:
This also needs to be added into GitLab but is currently blocked by #1408 !  Also see https://github.com/osbuild/osbuild-composer/pull/1405#discussion_r630822032

Reply:
This is now outdated. The test case is executed from the regression.sh script which in turn is executed from Gitlab CI configuration.